### PR TITLE
[Rust Server] Bugfix #5954 (Not support operationId-blank separator issue.)

### PR DESCRIPTION
**Pull request by paladinzh**
#5954 [BUG] [Rust Server] Not support operationId-blank separator issue.

### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) before.
- [x] Run the shell script(s) under `./bin/` (or Windows batch scripts under`.\bin\windows`) to update Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit, and these must match the expectations made by your contribution. You only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the code or mustache templates for a language (`{LANG}`) (e.g. php, ruby, python, etc).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.3.x`, `5.0.x`. Default: `master`.
- [x] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### CODE REVIEW

**Comment by wing328** [comment]
cc @frol (2017/07) @farcaller (2017/08) @bjgill (2017/12) @richardwhiuk (2019/07)

**Comment by wing328** [comment]
@paladinzh thanks for the PR, can you please merge the latest master of https://github.com/OpenAPITools/openapi-generator into the branch to see if the CI issues are resolved?

File: modules/openapi-generator/src/main/resources/rust-server/README.mustache [hunk 1]
```diff
@@ -109,7 +109,7 @@ All URIs are relative to *{{{basePath}}}*
 
 Method | HTTP request | Description
 ------------- | ------------- | -------------
-{{#apiInfo}}{{#apis}}{{#operations}}{{#operation}}[**{{{operationIdOriginal}}}**]({{{apiDocPath}}}{{classname}}_api.md#{{{operationIdOriginal}}}) | **{{{httpMethod}}}** {{{path}}} | {{#summary}}{{{summary}}}{{/summary}}
+{{#apiInfo}}{{#apis}}{{#operations}}{{#operation}}[**{{{operationId}}}**]({{{apiDocPath}}}{{classname}}_api.md#{{{operationId}}}) | **{{{httpMethod}}}** {{{path}}} | {{#summary}}{{{summary}}}{{/summary}}
```
Comment (richardwhiuk) [inline modules/openapi-generator/src/main/resources/rust-server/README.mustache:112]: I think we should keep the first one as `operationIdOriginal`, as that's what the user is going to expect to see?

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustServerCodegen.java [hunk 2]
```diff
@@ -441,7 +441,7 @@ public String toOperationId(String operationId) {
             operationId = "call_" + operationId;
         }
 
-        return camelize(operationId);
+        return camelize(operationId).replace(" ", "");
```
Comment (richardwhiuk) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustServerCodegen.java:444]: I'm surprised that camelize doesn't do this for us.

Looking at the code ( https://github.com/OpenAPITools/openapi-generator/blob/0fc2df6a4714343a9df621c8a5e6b767fe81a2cf/modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/StringUtils.java ), I think we might be better doing:

`return camelize(underscore(operationId));`

which I think will handle both issues.

File: modules/openapi-generator/src/main/resources/rust-server/README.mustache (hunk 1 above)
Comment (paladinzh) [reply modules/openapi-generator/src/main/resources/rust-server/README.mustache]: OK

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustServerCodegen.java (hunk 2 above)
Comment (paladinzh) [reply modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustServerCodegen.java:444]: Using camelize(underscore(operationId)) is not oK.
The example case will get the method "GetRepoinfo", Not the wanted "GetRepoInfo".

 **CreateRepo**](repo_api.md#CreateRepo) | **POST** /repos | 
-**GetRepoInfo**](repo_api.md#GetRepoInfo) | **GET** /repos/{repoId} | 
+**GetRepoinfo**](repo_api.md#GetRepoinfo) | **GET** /repos/{repoId} |

**Pull request closed without merging**